### PR TITLE
Added C++ playground

### PR DESCRIPTION
**Pull request by Ash515**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by Ash515** [comment]
Please give a result fast

**Comment by SethiPandi** [comment]
> Please give a result fast

Hey! It's not my place to comment really, but this repository is getting absolutely _flooded_ with PRs at the moment.
Despite this the maintainers are merging requests every single day pretty much, which takes a lot of time.
Free Programming Books is a _free_ resource and maintained by a _non-profit_.

If you're looking to finish Hacktoberfest ASAP, which I understand, rather than applying further pressure on maintainers, it might be better to explore more projects and make pull requests so there is a high chance they'll be approved/merged before October is over.

The easiest way to go would be to head over to https://github.com/topics/hacktoberfest and scroll through the list for as long as you want, starring any project that looks interesting to you. Afterwards, you can check them all out from your profile (https://github.com/Ash515?tab=stars) and see if there's anything you feel you can do. 

Good luck!

File: free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -99,6 +100,11 @@
 * [Try Erlang](http://www.tryerlang.org)
 
 
+### Dart
+
+* [Dart tutorials and course](https://hackr.io/tutorials/learn-dart) - Hackr.io
```
Comment (eshellman) [inline free-programming-interactive-tutorials-en.md:105]: This is a list of tutorials, not a tutorial, remove it.

File: free-programming-playgrounds.md [hunk 2]
```diff
@@ -42,6 +43,11 @@
 * [StackBlitz](https://stackblitz.com/fork/angular)
 
 
+### C++
+
+* [Repl.it](https://repl.it/@jturnerallen/C-Playground)
```
Comment (eshellman) [inline free-programming-playgrounds.md:48]: ok

**Pull request closed without merging**